### DISCUSSION TITLE
Instructions for installing Wavelog on a hosted server

### DISCUSSION THREAD

**Discussion by m0sdb** [wavelog/wavelog#166] [General]
Hi all

I wanted to install Wavelog on a hosted web server which did not allow for and scripting so I could not follow the instructions on the Wiki.

Instead, I just downloaded the zip of the whole thing from Github, unzipped it locally and then used an FTP program to upload the whole thing to my server.

When I when first went to index.php it happily ran the set up routine and everything is working beautifully.

Assuming there is nothing wrong with this approach would you like me to write up some instructions for people who need to install this way as a lot of hosting services do not allow scripting.  And apologies if these instructions are already done - I just couldn't find them in that case hihi.

My only question would be how I would install any updates using this method.

Vy 73 de Danny M0SDB

**Comment by HB9HIL** [comment]
Hello @m0sdb 

you are right. The wiki has some outdated stuff and some missing parts. So if you are motived and you know Wavelog then you are welcome to write some Wiki articles :)

**Reply by HB9HIL** [reply]
The Wiki is now open for everyone to edit :)

**Reply by dg9vh** [reply]
tnx for doing this... maybe I will throw my 2 cent into this when necessary :-)
